lib/config/default.js

```diff
@@ -1208,6 +1208,47 @@ defaultConfig.definition = () => ({
       formatter: boolean,
       default: false
     }
+  },
+
+  /**
+   * Obfuscates URL parameters
+   * for outgoing and incoming requests
+   * for distrubuted tracing attributes - both transaction and span attributes
+   * for transaction trace transaction details
+   */
+  url_obfuscation: {
```

Comment:
When this lands can you also open a PR in docs-website that describes these new configuration options?